Source/BusinessLogic/Logic/Players/PlayerRetriever.cs

```diff
@@ -54,24 +54,32 @@ public List<Player> GetAllPlayers(int gamingGroupId)
-                    .Include(player => player.PlayerGameResults)
-                        select new PlayerWithNemesis
-                        {
-                           PlayerId = player.Id,
-                           PlayerName = player.Name,
-                           PlayerRegistered = !string.IsNullOrEmpty(player.ApplicationUserId),
-                           NemesisPlayerId = player.Nemesis == null ? (int?)null : player.Nemesis.NemesisPlayerId,
-                           NemesisPlayerName = player.Nemesis != null && player.Nemesis.NemesisPlayer != null 
-                            ? player.Nemesis.NemesisPlayer.Name : null,
-                           PreviousNemesisPlayerId = player.PreviousNemesis == null ? (int?)null : player.PreviousNemesis.NemesisPlayerId,
-                           PreviousNemesisPlayerName = player.PreviousNemesis != null && player.PreviousNemesis.NemesisPlayer != null 
-                            ? player.PreviousNemesis.NemesisPlayer.Name : null,
-                            GamingGroupId = player.GamingGroupId,
-                            NumberOfPlayedGames = player.PlayerGameResults.Count
-                        }
-                   ).OrderBy(playerWithNemesis => playerWithNemesis.PlayerName)
-                   .ToList();
+            var playersWithNemesis = (from Player player in GetAllPlayersInGamingGroupQueryable(gamingGroupId)
+                .Include(player => player.PlayerGameResults)
+                select new PlayerWithNemesis
+                {
+                    PlayerId = player.Id,
+                    PlayerName = player.Name,
+                    PlayerRegistered = !string.IsNullOrEmpty(player.ApplicationUserId),
+                    NemesisPlayerId = player.Nemesis == null ? (int?)null : player.Nemesis.NemesisPlayerId,
+                    NemesisPlayerName = player.Nemesis != null && player.Nemesis.NemesisPlayer != null
+                        ? player.Nemesis.NemesisPlayer.Name : null,
+                    PreviousNemesisPlayerId = player.PreviousNemesis == null ? (int?)null : player.PreviousNemesis.NemesisPlayerId,
+                    PreviousNemesisPlayerName = player.PreviousNemesis != null && player.PreviousNemesis.NemesisPlayer != null
+                        ? player.PreviousNemesis.NemesisPlayer.Name : null,
+                    GamingGroupId = player.GamingGroupId,
+                    NumberOfPlayedGames = player.PlayerGameResults.Count,
+                    TotalPoints = player.PlayerGameResults.Select(pgr => pgr.NemeStatsPointsAwarded).DefaultIfEmpty(0).Sum()
+                }
+                ).OrderByDescending(pwn => pwn.TotalPoints).ThenBy(p => p.PlayerName)
+                .ToList();
+
+            foreach (var playerWithNemesis in playersWithNemesis)
+            {
+                playerWithNemesis.Championships = GetChampionedGames(playerWithNemesis.PlayerId);
```

Comment:
If we end up only showing the number of championed games then we can just move this into the projection and do a count there.